template.yaml

```diff
@@ -0,0 +1,956 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >-
+  Template to create the infrastructure for the Auth Relying Party stub.
+
+Transform:
+  - AWS::Serverless-2016-10-31
+  - AWS::LanguageExtensions
+
+Parameters:
+  VpcStackName:
+    Description: "The name of the stack that defines the VPC that this container will run in"
+    Type: "String"
+  Environment:
+    Description: "Environment the infrastructure is deployed to"
+    Type: "String"
+
+Conditions:
+  IsProd:
+    !Equals [ production, !Ref Environment]
+
+Mappings:
+  EnvironmentConfiguration:
+    build:
+      baseUrl: "https://oidc.build.account.gov.uk/"
+    staging:
+      baseUrl: "https://oidc.staging.account.gov.uk/"
+    integration:
+      baseUrl: "https://oidc.integration.account.gov.uk/"
+    production:
+      baseUrl: "https://oidc.account.gov.uk/"
+
+Resources:
+  #
+  # Fargate cluster
+  #
+  FargateCluster:
+    Type: AWS::ECS::Cluster
+    Properties:
+      ClusterSettings:
+        - Name: containerInsights
+          Value: enabled
+      Tags:
+        - Key: Name
+          Value: !Sub "${AWS::StackName}-FargateCluster"
+        - Key: Service
+          Value: Relying Party stub
+        - Key: Source
```

Comment:
Probably good to tag this with `Owner: di-orchestration@digital.cabinet-office.gov.uk`